recipe/meta.yaml

```diff
@@ -38,7 +38,8 @@ requirements:
     - fire
     - geopy
     - humanfriendly
-    - iris >=3.4.0
+    - importlib_metadata
+    - iris >=3.6.0
```

Comment:
I'm not entirely sure about this one, see https://github.com/ESMValGroup/ESMValTool/issues/3239#issuecomment-1613298587

Reply:
we should mirror what we have in our env upstream

Reply:
On the other hand, stuff is increasingly likely to not work with older iris versions..

Reply:
I am pretty sure we require at least iris v3.5, looking for the PR as we speak

Reply:
yep 3.4 was a bad apple 🍏

Reply:
Found it: https://github.com/ESMValGroup/ESMValCore/pull/2139. This requires v3.5, which was actually bugfixed in v3.6.1. So technically, we need to pin to iris>3.6.1

Reply:
which is fine since it'll grab 3.8 etc - but defo pin on higher version, and not 3.4

Reply:
I'm going to set it to 3.6.1 because versions 3.5 and 3.6 had the issue with the weights being realized: https://github.com/SciTools/iris/issues/5338

Reply:
We should probably do the same for our environment and setup.py

Reply:
make it so (in Jean Luc Picard voice)

Reply:
Let's open an issue about that

Reply:
see https://github.com/ESMValGroup/ESMValCore/issues/2285